### PR TITLE
Skills multiselect

### PR DESCRIPTION
**Pull request by EHale64**
added the multi-select tool for a users' skills

### CODE REVIEW

File: app/models/skill.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Skill < ApplicationRecord
```
Comment (twarbelow) [inline app/models/skill.rb:1]: please document somewhere how exactly the skill table will add to functionality in the future (we will use this for our readme)

Comment (twarbelow) [inline app/models/skill.rb:1]: I'm not seeing a model spec for Skill...

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -11,7 +11,17 @@
    <%= label_tag :username %>
    <%= text_field_tag :username, current_user.username %>
    <%= label_tag :about_me %>
-   <%= text_field_tag :about_me %>
+   <%= text_field_tag :about_me, current_user.about_me %>
+   <%= label_tag :role %>
+   <%= select_tag :role, options_for_select(['Teacher', 'Student', 'Performer']), multiple: true %>
+   <%= label_tag :skills %>
+   <%= select_tag :skills, options_for_select(['Solo', 'Chamber', 'orchestral', 'pedagogy', 'complex rhythms', 'extended technique' ]), multiple: true %>
```
Comment (twarbelow) [inline app/views/users/edit.html.erb]: I know Kwibe mentioned we will solve the capitalization in frontend, but this looks messy with the differences in capitalization.

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -12,10 +12,20 @@
 
 </section>
 
+<% if current_user.skills.size > 0 %>
 <section class="my-skills">
-
+  <h3>My Skills:</h3>
+  <ul>
+    <% current_user.skills.each do |skill| %>
+    <li><%= skill.skill %></li>
```
Comment (twarbelow) [inline app/views/users/show.html.erb]: skill.skill is....weird looking

Comment (twarbelow) [reply app/views/users/show.html.erb]: but fine?

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -12,10 +12,20 @@
...
+    <% end %>
+  </ul>
 </section>
-
+<% else %>
+ <section class='my-skills'>
+  <p>You currently have no skills</p>
```
Comment (twarbelow) [inline app/views/users/show.html.erb]: can we change this to "You have not added any skills." to be more polite?

**Review by twarbelow — CHANGES_REQUESTED** [review]

File: app/views/users/edit.html.erb (hunk 2 above)
Comment (EHale64) [reply app/views/users/edit.html.erb]: Copy that. I'll take a minute to change it to be consistent

File: app/views/users/show.html.erb (hunk 3 above)
Comment (EHale64) [reply app/views/users/show.html.erb]: might change the first skill to be `user_skill` for readability

File: app/views/users/show.html.erb (hunk 4 above)
Comment (EHale64) [reply app/views/users/show.html.erb]: Sure

File: app/models/skill.rb (hunk 1 above)
Comment (EHale64) [reply app/models/skill.rb:1]: it should be `skill_spec.rb` seeing it on this end.

Comment (EHale64) [reply app/models/skill.rb:1]: Added a comment on line 3 of `skill.rb`

File: app/views/users/show.html.erb (hunk 4 above)
Comment (jKwibe) [reply app/views/users/show.html.erb]: Nice catch @twarbelow

File: app/models/skill.rb (hunk 1 above)
Comment (jKwibe) [reply app/models/skill.rb:1]: @ckccameron @twarbelow  @EHale64 There are no tests for the skills model. 
Note: We didn't configure simplecov so it is hard to see our test coverage.

File: app/views/users/show.html.erb (hunk 3 above)
Comment (twarbelow) [reply app/views/users/show.html.erb]: that would work

File: app/models/skill.rb (hunk 1 above)
Comment (twarbelow) [reply app/models/skill.rb:1]: please add a card in our todo to add simplecov

**Review by twarbelow — APPROVED** [review]

**Pull request merged**